e2e/Dockerfile.jenkins

```diff
@@ -1,7 +1,5 @@
-FROM ubuntu:20.04
-
-# Setting bash as the default shell
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+ARG NODE_VERSION=16.15.0
```

Comment:
A simpler Dockerfile using the official node docker images, rather than nvm, the NODE_VERSION is set as part of the `docker-build.sh` by reading the version set in the `.npmrc`

Reply:
I love that we are removing nvm! 👍 What do you think about not providing the default value for `NODE_VERSION` here? I imagined, in a few months, someone might think that we need to update node version in both `.nvmrc` and this file If they don't know `docker-build.sh` is reading the version set in the `.npmrc`

Reply:
The idea of build args for Docker is to override them when running `docker build`. If it causes any issues, we can remove it, no strong opinion.

Reply:
```suggestion
ARG NODE_VERSION
```